cnn/cnn.css

```diff
@@ -0,0 +1,62 @@
+.containr {
+    width: 585px;
+    padding: 20px;
+    border: 1px solid gray;
+    margin-bottom: 10px;
+    border-radius: 5px;
+}
+
+.ArticleIMG {
+    width: 85px;
+    height: 85px;
+    background-size: 85px;
+    background-repeat: no-repeat;
+    background-image: url('http://www.cs.cmu.edu/~rsinha/Courses/45-934/ant2.jpg');
+    float: right;
+}
+
+.article {
+    width: 500px;
+    float: left;
+}
+
+.Icon {
+    width: 20px;
+    height: 20px;
+    background-image: url('https://www.shareicon.net/data/128x128/2016/06/21/611593_article_128x128.png');
+    background-size: 20px;
+    float: left;
```

Comment:
I wouldn't float:left the items on the left. Instead I would use display:inline or display:inline-block, which still leaves them on the left (less complexity).

That said, I'd leave the menu using float:right (and would still need the `clear`).